### DISCUSSION TITLE
每次重启网易云后之前对插件的设置都会被初始化

### DISCUSSION THREAD

**Discussion by dlhaizhe2023** [amll-dev/applemusic-like-lyrics#214] [Q&A]
请问有人遇到这个问题吗？我每次打开网易云时这个插件都会弹出新手教程，并且我之前的设置也都没有保存，不知有没有解决方法
/(ㄒoㄒ)/~~

**Comment by Steve-xmh** [comment]
你需要检查一下自己插件文件夹的访问权限，有可能是权限不足导致不能写入配置文件

**Comment by Linho1219** [comment]
AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

现关闭此讨论，敬请谅解。
